ArchiSteamFarm/Steam/Data/EAssetRarity.cs

```diff
@@ -20,12 +20,25 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
 
 namespace ArchiSteamFarm.Steam.Data;
 
+#pragma warning disable CA1027 // Aliases are intentional, we don't plan to combine fields
 public enum EAssetRarity : byte {
 	Unknown,
 	Common,
 	Uncommon,
-	Rare
+	Rare,
+	Mythical,
```

Comment:
I think we should document the non-trading-card rarities and which game(s) they are used in. Do you mind adding some comments?

Reply:
Not needed imho.